pkg/kubectl/cmd/convert.go

```diff
@@ -78,55 +99,17 @@ func NewCmdConvert(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *co
 
 	options.PrintFlags.AddFlags(cmd)
 
-	usage := "to need to get converted."
-	cmdutil.AddFilenameOptionFlags(cmd, &options.FilenameOptions, usage)
-	cmd.MarkFlagRequired("filename")
 	cmdutil.AddValidateFlags(cmd)
+	cmdutil.AddFilenameOptionFlags(cmd, &options.FilenameOptions, "to need to get converted.")
+	cmd.MarkFlagRequired("filename")
+
 	cmd.Flags().BoolVar(&options.local, "local", options.local, "If true, convert will NOT try to contact api-server but run locally.")
-	cmd.Flags().String("output-version", "", i18n.T("Output the formatted object with the given group version (for ex: 'extensions/v1beta1').)"))
+	cmd.Flags().StringVar(&options.OutputVersion, "output-version", "", i18n.T("Output the formatted object with the given group version (for ex: 'extensions/v1beta1').)"))
```

Comment:
Use `o` instead of `options`, and wire the default value from `o.OutputVersion` as well.